src/main/java/com/mide/gangsaeng/board/CacheableBoardRepository.java

```diff
@@ -34,6 +33,15 @@ public void write(Board board) {
         cache.write(boardDataToBeStored);
     }
 
+    @Override
```

Comment:
Question.
What will happen if we add transaction(`@Transactional`) for update here?
Give me the scenario

Reply:
Then what will happen if we do not catch exceptions in write method for redis implementation?

> If we want to use transactions with a Redis client, we should use other commands or Spring Data Redis with additional settings.
What does it mean?

Reply:
> Then what will happen if we do not catch exceptions in write method for redis implementation?
Stale data might be served to users.


> > If we want to use transactions with a Redis client, we should use other commands or Spring Data Redis with additional settings.
> 
> What does it mean?

It means if we want to use transactions in Redis(lettuce library), using only `@Transactional`annotation is not enough. For example, we can use `multi()` to enter transactions and `exec()` to execute all commands.
[https://lettuce.io/core/release/reference/index.html#_transactionsmulti](url)

In Spring Data Redis, we can use `@Transactional` after setting RedisTransactionManager or use `SessionCallback` with Redis commands.
[https://docs.spring.io/spring-data/redis/reference/redis/transactions.html](url)